app/authenticated/project/api/hooks/index/controller.js

```diff
@@ -4,7 +4,10 @@ import Sortable from 'ui/mixins/sortable';
 export default Ember.Controller.extend(Sortable, {
   settings: Ember.inject.service(),
 
-  sortableContent: Ember.computed.alias('model.receivers'),
+  sortableContent: function() {
+    let receivers = this.get('model.receivers');
+    return receivers.filter(ele => ele.driver !== 'forwardPost');
+  }.property('model.receivers.@each'),
```

Comment:
@each needs a key name after it, just @each is invalid.  Or you meant `model.receivers.[]`, but it should be `model.receivers.@each.driver` here.

Reply:
Sorry for that, I meant to `model.receivers.[]`, but use it wrong, I thought the property `driver` won't be changed. 
But, I think your are right, using `model.receivers.@each.driver` is more accurate, I've updated to `model.receivers.@each.driver`.